xs2a-adapter-aspsp-registry/src/test/java/de/adorsys/xs2a/adapter/registry/LuceneAspspRepositoryTest.java

```diff
@@ -14,6 +16,57 @@
 
     LuceneAspspRepository luceneAspspRepository = new LuceneAspspRepository(new ByteBuffersDirectory());
 
+    @Test
+    public void create() {
+        Aspsp aspsp = new Aspsp();
+        luceneAspspRepository.save(aspsp);
+
+        aspsp.setId("1");
+        luceneAspspRepository.create(aspsp);
+    }
+
+    @Test(expected = DuplicationAspspException.class)
+    public void create_Duplication() {
+        Aspsp aspsp = new Aspsp();
+        luceneAspspRepository.save(aspsp);
+
+        aspsp.setId("1");
+        luceneAspspRepository.create(aspsp);
+
+        luceneAspspRepository.create(aspsp);
+    }
+
+    @Test
+    public void delete() {
```

Comment:
This and the following test don't assert anything.

Reply:
thanks, fixed